lib/compile-exports.js

```diff
@@ -11,19 +11,26 @@ module.exports = function compileExports(result, importItemMatcher, camelCaseKey
     return "";
   }
 
+  var symbols = [];
   var exportJs = Object.keys(result.exports).reduce(function(res, key) {
     var valueAsString = JSON.stringify(result.exports[key]);
     valueAsString = valueAsString.replace(result.importItemRegExpG, importItemMatcher);
-    res.push("\t" + JSON.stringify(key) + ": " + valueAsString);
+
+    camelCaseKey = camelCase(key);
 
     if (camelCaseKeys === true) {
-      res.push("\t" + JSON.stringify(camelCase(key)) + ": " + valueAsString);
+      symbols.push(camelCaseKey + ": " + camelCaseKey);
     } else if (camelCaseKeys === 'dashes') {
-      res.push("\t" + JSON.stringify(dashesCamelCase(key)) + ": " + valueAsString);
-    }
+      camelCaseKey = dashesCamelCase(key);
+      symbols.push(camelCaseKey + ": " + camelCaseKey);
+    }     
+
+    res.push("export const " + camelCaseKey + " = " + valueAsString);
+    symbols.push("'" + key + "': " + camelCaseKey);
 
     return res;
-  }, []).join(",\n");
+  }, []);
+  exportJs.push("export default { " + symbols.join(", ") + " };");
```

Comment:
It does both, exports each separate symbol but also exports all symbols as properties on one default object.

This was done for compatibility with the CSS modules spec where eg `import styles from './styles.css';" is expected to give you such an object. Exporting the CSS source by default makes more sense in terms of a loader but wouldn't allow that behaviour. It also allows dashed-case inside the default object, although not in the separated symbols.